jenkins/bisect.jpl

```diff
@@ -390,16 +390,32 @@ def findMergeBase(kdir, good, bad) {
             script: "git merge-base --is-ancestor ${base} HEAD")
 
         if (good_base != 0) {
-            def ref = "${params.REF_KERNEL_TREE}/${params.REF_KERNEL_BRANCH}"
+
+            dir(kci_core) {
```

Comment:
You could put `dir(kci_core)` inside the `if(...)` block as there's no need to change directory to do that.

Reply:
Changed